GOVERNANCE.md

```diff
@@ -124,11 +127,30 @@ the issue tracker is:
 
 ## Collaborator nominations
 
-Existing collaborators can nominate someone to become a collaborator. Nominees
-should have significant and valuable contributions across the Node.js
+### Who can nominate Collaborators?
+
+Existing Collaborators can nominate someone to become a Collaborator.
+
+### Ideal Nominees
+
+Nominees should have significant and valuable contributions across the Node.js
 organization.
 
-To nominate a new collaborator, open an issue in the [nodejs/node][] repository.
+Contributions can be:
+
+* Commits
+* Pull requests
+* Comments
+* Reviews
+* Helping others
```

Comment:
I would remove this, because IMO all contributions are "helping" (and similarly, it's possible to be a very helpful person without contributing to the project)
```suggestion
```